odoo/addons/test_translation_import/tests/test_term_count.py

```diff
@@ -348,7 +348,12 @@ def test_export_import(self):
         """ Ensure export+import gives the same result as loading a language """
         # load language and generate missing terms to create missing empty terms
         with mute_logger('odoo.addons.base.models.ir_translation'):
-            self.env["base.language.install"].create({'lang': 'fr_FR', 'overwrite': True}).lang_install()
+            self.env['res.lang']._activate_lang('fr_FR')
```

Comment:
Strange you have to activate it, was not the case before ... maybe simply search the lang with an active_test flag herebelow ?

Reply:
Looks like with context on the `lang_ids` this works

Reply:
Never mind, looks like activating the language is needed.

Reply:
if the goal is to get the id of `fr_FR`, indeed no need to activate it, just use `env.ref('base.lang_fr).id`

Reply:
Done :)